code/game/machinery/suit_storage_unit.dm

```diff
@@ -219,10 +226,15 @@
 			// The wires get damaged too.

```

Comment:
can you fix a little thing above this? the code is calling qdel() for each item inside but it nullifies the reference beforehand so it currently doesn't qdel() anything. It should be qdel(X) and THEN X = null